src/pudl/analysis/record_linkage/__init__.py

```diff
@@ -0,0 +1 @@
+"""This module impolements models for various forms of record linkage."""
```

Comment:
The pattern we've adopted in the other subpackages is to have each `__init__.py` import all of the modules in that subpackage.  Should we do that here? Or should we be doing something different everywhere?

Reply:
I don't think that the imports with `__init__.py`, are strictly necessary, and in my experience it's pretty common practice to have empty `__init__.py` files in sub-packages. I always thought it was more if you want to make code from a module available in the package level namespace you could import it in `__init__.py`, but python doesn't have a problem finding modules within a package. All that said, I don't really feel like I have a firm grasp on import best practices, and I'm totally fine with sticking with the pattern used elsewhere for consistency.